README.md

```diff
@@ -1,56 +1,31 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Discover Crypto
 
-> Description the project.
+> A crypto market app that showcases different top coins with their respective info, built using React and styled with pure CSS. the info is fetched from CoinGecko website.
 
+Check out a demo explanation about this project here [Live Video Demo Explainer](https://www.loom.com/share/0c43fbd5ba9b448ebd97848be08f17c5)
 
 ## Built With
 
```

Comment:
- [ ] Your README is lacking two important sections. First is the `screenshot section` This is where you add the screenshot of your web app to give visitors a quick preview of what your app looks like 😃 you understand that it is very important. The second is `getting started section` How can someone who wants to use your application locally do it? what procedures to follow? what are the prerequisites? how to run the application locally? how to run tests? these are all questions that should be answered in this section so please add those two sections 😉

Reply:
@BertrandConxy That's true, thank you for the feedback.